ui/scroll-parent.js

```diff
@@ -0,0 +1,43 @@
+/*!
+ * jQuery UI Scroll Parent @VERSION
+ * http://jqueryui.com
+ *
+ * Copyright jQuery Foundation and other contributors
+ * Released under the MIT license.
+ * http://jquery.org/license
+ */
+
+//>>label: scrollParent
+//>>group: Core
+//>>description: Get the closest ancestor element that is scrollable.
+//>>docs: http://api.jqueryui.com/category/scrollParent/
+//>>demos: http://jqueryui.com/scrollParent/
```

Comment:
These don't exist. And I doubt that we'll write a demo for this. API docs seem fine, but it won't be a category.